frb_rust/src/ffi/mod.rs

```diff
@@ -55,3 +57,28 @@ pub fn wire2api_uuids(ids: Vec<u8>) -> Vec<uuid::Uuid> {
         .map(wire2api_uuid_ref)
         .collect::<Vec<uuid::Uuid>>()
 }
+
+/// Equivalent to a [`Arc::clone()`], but directly in terms of raw pointers.
+///
+/// # Safety
+///
+/// This function should never be called manually.
+extern "C" fn share_arc<T>(ptr: *const c_void) -> *const c_void {
```

Comment:
https://api.dart.dev/stable/2.18.4/dart-ffi/NativeFunction-class.html
as I understand it for ffi.NativeFunction (native implementation) requires `extern C`, in case of web implementation it is not required

Reply:
Oh I see https://github.com/fzyzcjy/flutter_rust_bridge/pull/795/files/6dda6097df034a6a78dacc541770b6722906c8b8#diff-2c1bb60c0a7027260e20af8c5501d9f7d0131a267b89572ea4e720e1317d3813R100. You are right